channel/funder_test.go

```diff
@@ -396,23 +397,23 @@ func newNFunders(
 	simBackend.FundAddress(ctx, tokenAcc.Address)
 	cb := ethchannel.NewContractBackend(
 		simBackend,
-		keystore.NewTransactor(*ksWallet, test.SimSigner),
+		keystore.NewTransactor(*ksWallet, test.SignerForChainID(chainID)),
```

Comment:
Because the signer is dependent of the chainID which may vary during the different usages of the signer. 
Before, the signer was static and always had 1337 as its chainID.